canonical-websites/www.ubuntu.com
# Temp new /legal/ubuntu-advantage-service-description/new page

## Done

- Created a new version of the UASD for sales to refer to until it goes live

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/legal/ubuntu-advantage-service-description/new](http://0.0.0.0:8001/legal/ubuntu-advantage-service-description/new)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc](https://docs.google.com/document/d/1vQNdVmpsB3iQP7SnJHxT3A8rbD5jSAzTEoYUOepR-SM/edit#heading=h.g259y66d4s5e)


## Issue / Card

Fixes #4669
